lib/results-converter.js

```diff
@@ -27,14 +27,35 @@ const matchesToString = (matches, indent) => {
 	}, '');
 };
 
+const isFileCollapsed = (filePath) => {
+	const selectorParent = '.list-nested-item.collapsed';
+	const selectorListItem = `[data-file-path="${filePath}"]`;
+  try {
+    const res = atom.document.querySelectorAll(`${selectorParent} ${selectorListItem}`);
+    return res.length > 0;
+  } catch(e) {
+    return false;
+  }
+	return false;
```

Comment:
nitpick: fix indentation

Reply:
![omg wtf](https://media1.giphy.com/media/3o7TKnYVvhjh6qk0VO/giphy.gif)

Reply:
Thanks for the favor 👍